zerver/openapi/zulip.yaml

```diff
@@ -19869,6 +19869,24 @@ paths:
             **Note**: This endpoint is only available to [members and
             administrators](/help/roles-and-permissions); bots and guests
             cannot use this endpoint.
+      requestBody:
+        required: false
+        content:
+          application/x-www-form-urlencoded:
+            schema:
+              type: object
+              properties:
+                allow_deactivated:
+                  description: |
+                    Whether to include deactivated user groups in the response.
+
+                    **Changes**: New in Zulip 10.0 (feature level 285).
```

Comment:
This should just say "Previously, deactivated user groups did not exist and thus would enver be included in the response."?

Reply:
Updated.